lua/starfall/libs_sh/hologram.lua

```diff
@@ -238,6 +238,38 @@ else
 			sfParent:updateTransform()
 		end
 	end
+	
+	--- Sets the hologram's position local to its parent.
+	-- @shared
+	-- @param Vector vec New position
+	function hologram_methods:setLocalPos(vec)
+		local holo = getholo(self)
+		local pos = SF.clampPos(vunwrap(vec))
+		checkpermission(instance, holo, "hologram.setRenderProperty")
+
+		holo:SetLocalPos(pos)
+
+		local sfParent = holo.sfParent
+		if sfParent and IsValid(sfParent.parent) then
+			sfParent:updateTransform()
+		end
+	end
+
+	--- Sets the hologram's angles local to its parent.
+	-- @shared
+	-- @param Angle ang New angles
+	function hologram_methods:setLocalAngles(ang)
+		local holo = getholo(self)
+		local angle = aunwrap(ang)
+		checkpermission(instance, holo, "hologram.setRenderProperty")
+
+		holo:SetLocalAngles(angle)
+		
+		local sfParent = holo.sfParent
+		if sfParent and IsValid(sfParent.parent) then
+			sfParent:updateTransform()
+		end
+	end
```

Comment:
Holograms should just use the regular entity setLocalPos setLocalAngles. the sfParent code should be added to those.

Reply:
The current arrangement exactly mirrors what is done with Entity:setPos() and Entity:setAngles() and Hologram:setPos() and Hologram:setAngles(). The entity methods and their new local counterparts are serverside only, while the hologram methods are shared. Unless you are suggesting that the Entity methods should be shared instead of server only, and have the proposed change extend to all of the Hologram methods, I don't really see why we would do this.